engine-cockpit-selenium-test/test/ch/ivyteam/enginecockpit/services/WebTestWebServiceProcesses.java

```diff
@@ -25,5 +32,11 @@ void beforeEach() {
   @Test
   void webServices() {
     Tab.APP.switchToDefault();
+    $("h2").shouldHave(text("Web Service Processes"));
+
+    $$("[id$=':webServiceProcessesTable_data'] tr a").shouldHave(size(1));
+
+    var table = PrimeUi.table(By.cssSelector("[id$=':webServiceProcessesTable']"));
```

Comment:
I would just put here the whole id instead of this matcher...
Best use the By.id() selector then you don't have to escape the ":"